portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -20,6 +20,7 @@
 import com.google.appengine.api.datastore.PreparedQuery;
 import com.google.appengine.api.datastore.Query;
 import com.google.appengine.api.datastore.Query.SortDirection;
+import com.google.appengine.api.datastore.FetchOptions;
```

Comment:
Alphabetize the imports (this is in the style guide internally so probably not know to interns)